code/script.js

```diff
@@ -204,6 +204,9 @@ const CHARACTERS = [
 let secret
 let currentQuestion
 let charactersInPlay
+let category
+let value
+let keep = true;
```

Comment:
I'm curious about the use of semicolons in this section. Did you have something specific in mind when adding a semicolon in line 209, but not to the rest of the global variables? 

This is a really minor detail, nothing that affects the code performance or such :) But for style consistency, I think there could be two options: 

```
let secret
let currentQuestion
let charactersInPlay
let category
let value
let keep = true

```

Or (I tend to use this one, but it's really a matter of taste): 

```
let secret, 
    currentQuestion, 
    charactersInPlay, 
    category, 
    value, 
    keep = true;
```

Maybe using the Prettier extension on VS Code helps, too?